airbyte-integrations/connectors/destination-clickhouse/build.gradle

```diff
@@ -32,7 +32,7 @@ application {
 }
 
 dependencies {
-    implementation 'com.clickhouse:client-v2:0.9.0'
+    implementation 'com.clickhouse:client-v2:0.9.4'
```

Comment:
upgrading clickhouse client to pull in bugfix https://github.com/ClickHouse/clickhouse-java/issues/2359

otherwise we can't read JSON columns